packages/ic-response-verification/src/validation/v2_validation.rs

```diff
@@ -2,45 +2,27 @@ use crate::{ResponseVerificationError, ResponseVerificationResult};
 use ic_certification::hash_tree::HashTreeNode;
 use ic_certification::{hash_tree::Hash, HashTree, Label, SubtreeLookupResult};
 use ic_http_certification::cel::DefaultCelExpression;
+use ic_http_certification::utils::{
+    is_wildcard_path_valid_for_request_path, more_specific_wildcards_for,
+    EXACT_PATH_TERMINATOR_BYTES, PATH_PREFIX_BYTES,
+};
 use ic_http_certification::CelExpression;
 
-fn path_from_parts<T>(parts: &[T]) -> Vec<Label>
+fn path_from_parts<T>(parts: &[T]) -> Vec<Vec<u8>>
```

Comment:
There's a few changes here that are related to switching from `Label` to `Vec<u8>`. `Label` didn't have any advantage and `Vec<u8>` made it easier to share code with `ic-http-certification`.